src/main/java/mate/academy/bookshop/repository/UserRepository.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.bookshop.repository;
+
+import mate.academy.bookshop.model.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+
+public interface UserRepository extends JpaRepository<User, Long>, JpaSpecificationExecutor<User> {

```

Comment:
we do not need it here for now, and I do not think we will need it in the future
```suggestion
public interface UserRepository extends JpaRepository<User, Long> {
```